services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +43,32 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
+   xQueueSend(thermalMgrQueueHandle, event, 0); // why u throwing error
 
   return ERR_CODE_SUCCESS;
 }
 
 void osHandlerLM75BD(void) {
   /* Implement this function */
+  float temp;
+  readTempLM75BD(LM75BD_OBC_I2C_ADDR, &temp); 
+
+  if(temp > 80) {
+    overTemperatureDetected();
+  } else if(temp < 75) {
+    safeOperatingConditions();
+  }
```

Comment:
Do not read inside the os handler as it may block. Send an event and have the thermalMgr task function handle it